pkgs/development/tools/twitch-cli/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "twitch";
```

Comment:
```suggestion
  pname = "twitch-cli";
```

Reply:
When you type twitch -h you get the help/documentation: `Use "twitch [command] --help" for more information about a command.`. So i named it twitch on purpose.

Reply:
set meta.mainProgram = "twitch" and pname to "twitch-cli"